sensor/common/networkflow/manager/manager_impl.go

```diff
@@ -337,6 +337,7 @@ func (m *networkFlowManager) enrichConnections(tickerC <-chan time.Time) {
 				continue
 			}
 			m.enrichAndSend()
+			m.clusterEntities.Tick() // Used to measure whether historical endpoints should still be remembered
```

Comment:
It seems a bit odd to call this in `enrichConnections` as this does not just tick connections. The comment mentions endpoints, but the function name is about connections.